nextflow.config

```diff
@@ -389,6 +389,11 @@ process.shell = [
 // Disable process selector warnings by default. Use debug profile to enable warnings.
 nextflow.enable.configProcessNamesValidation = false
 
+// Enable loading modules bin files
+nextflow.enable.moduleBinaries = true
+
+def trace_timestamp = new java.util.Date().format( 'yyyy-MM-dd_HH-mm-ss')
```

Comment:
This change doesn't seem related? Is it here on purpose?

Reply:
Let me ask @alanmmobbs93 if it has some functionality, but I also think this should be deleted.

Reply:
`trace_timestamp` was a line from the old template. I've removed it now.